zokrates_core/src/proof_system/bn128/g16.rs

```diff
@@ -339,6 +342,7 @@ mod tests {
             use crate::ir::*;
             use crate::proof_system::bn128::g16::serialize::serialize_proof;
             use typed_absy::types::{Signature, Type};
+            use zokrates_field::Bn128Field;
```

Comment:
Should we test for all supported curves, at least for the proof systems?

Reply:
Added serialization test for bls12

Reply:
I added support for many curves for anything using `zokrates_test`